src/Settings/ExtensionSetting.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * Class representing a specific setting for the Teamscale Azure DevOps extension. Each setting object consists of
+ * the setting's key and the default value to be used in case the setting has not been configured.
+ */
+export class ExtensionSetting {
```

Comment:
thought: This looks like a good use case for generics so that we have actual, e.g., boolean settings. But I'd also be happy to leave it as is in ordner not to risk too much here, esp. since this might be another medium size change (we'd need to change the type of `configuredVssSettings` in `Settings` and figure out how to properly parse/store the values there). Also, I am not that familiar with generics in Typescript 🙈. So I'll leave that up to you 🙂

Reply:
I'm also not quite familiar and I would say that's out of scope for the urgent bug ticket. I's suggest creating a follow-up maintenance ticket for the refactoring, wdyt?